src/containerapp/azext_containerapp/custom.py

```diff
@@ -2863,7 +2863,10 @@ def create_session_pool(cmd,
                         target_port=None,
                         registry_server=None,
                         registry_pass=None,
-                        registry_user=None):
+                        registry_user=None,
+                        user_assigned=None,
+                        registry_identity_id=None,
```

Comment:
About `registry_identity_id`, should it be `registry_identity`? That will be consistent with other commands' registry identity.